deployHandler.go

```diff
@@ -66,12 +98,12 @@ func (dc *DeployCloudAppPayload) validate() ([]string, error) {
 }
 
 // NewDeployHandler creates a cloudApp controller.
-func NewDeployHandler(logger Logger, paasService PaaSService, namer Namespacer) DeployHandler {
+func NewDeployHandler(logger Logger, deployController controller.Deploy, namer Namespacer) DeployHandler {
 	return DeployHandler{
-		logger:      logger,
-		paasService: paasService,
-		namer:       namer,
-		objectNamer: rhmap.Service{},
+		logger:           logger,
+		deployController: deployController,
```

Comment:
yeah, I initially thought this was a nested deployController, I hadn't noticed the different names. If DeployHandler is the part that knows about HTTP, should that be reflected in it's name?